packages/client/src/utils/index.ts

```diff
@@ -148,20 +149,27 @@ export async function buildFileTree(
 
 export async function getVercelIgnore(
   cwd: string | string[],
-  prebuilt?: boolean
+  prebuilt?: boolean,
+  vercelOutputDir?: string
 ): Promise<{ ig: Ignore; ignores: string[] }> {
   const ig = ignore();
   let ignores: string[];
 
   if (prebuilt) {
-    const outputDir = '.vercel/output';
+    if (typeof vercelOutputDir !== 'string') {
+      throw new Error(`Missing required \`vercelOutputDir\` parameter`);
+    }
+    if (typeof cwd !== 'string') {
+      throw new Error(`\`cwd\` must be a "string"`);
+    }
+    const relOutputDir = relative(cwd, vercelOutputDir);
     ignores = ['*'];
-    const parts = outputDir.split('/');
+    const parts = relOutputDir.split(sep);
```

Comment:
Why does this use `sep` when the next two calls use raw `'/'`?

Reply:
For this (and your next comment), it's because the input is (potentially) a Windows path (with `\`), but the `ignore` module expects Unix paths